plugins/de.cau.cs.kieler.klighd/src/de/cau/cs/kieler/klighd/util/KlighdProperties.java

```diff
@@ -375,4 +375,11 @@ public static boolean isSelectable(final EObject viewElement) {
      */
     public static final IProperty<Boolean> IS_NODE_TITLE =
             new Property<Boolean>("klighd.isNodeTitle", false);
+    
+    /**
+     * Automatically computes the padding required to fit the content of a node within the bounds of a 
+     * rounded rectangle. The x and y corner radii are specified as x and y of a KVector.
+     */
```

Comment:
This documentation should include the interaction with the padding property, in that it overrides any existing padding on the element.